router/widen-lemma.gobra

```diff
@@ -136,15 +136,16 @@ func absPktWidenLemma(raw []byte, length int) {
 	seg1Len := int(metaHdr.SegLen[0])
 	seg2Len := int(metaHdr.SegLen[1])
 	seg3Len := int(metaHdr.SegLen[2])
-	segLen := scion.LengthOfCurrSeg(currHfIdx, seg1Len, seg2Len, seg3Len)
-	prevSegLen := scion.LengthOfPrevSeg(currHfIdx, seg1Len, seg2Len, seg3Len)
-	numINF := scion.NumInfoFields(seg1Len, seg2Len, seg3Len)
+	s := io.CombineSegLens(seg1Len, seg2Len, seg3Len)
```

Comment:
usually, I would call this variable `segLens` instead of `s` (this applies to many parts  of this PR)